src/Frontend/src/composables/serviceServiceControl.ts

```diff
@@ -8,6 +8,7 @@ import type RootUrls from "@/resources/RootUrls";
 import type FailedMessage from "@/resources/FailedMessage";
 import type MonitoredEndpoint from "@/resources/MonitoredEndpoint";
 import { FailedMessageStatus } from "@/resources/FailedMessage";
+import { useMonitoringStore } from "@/stores/MonitoringStore";
```

Comment:
This import can be removed
```suggestion
```

Reply:
Yep, that was left over from another idea I was trying first.